src/app/[locale]/viewer/_components/AASOverviewCard.tsx

```diff
@@ -209,10 +215,21 @@ export function AASOverviewCard(props: AASOverviewCardProps) {
                                             cursor: 'pointer',
                                         },
                                     }}
+                                    onErrorCapture={handleError}
                                 />
                             ) : (
-                                <StyledImage src={productImageUrl} />
-                            ))}
+                                <StyledImage src={productImageUrl} onErrorCapture={handleError} />
```

Comment:
Not sure, when the `onErrorCapture` is triggered, but right now it sometimes prevents the image from being loaded even if it should be loaded. 
(See this test aas: https://wgrp.biz/aas/xNBLy1C and all AAS in the Compare view)